[MCS-DE] Bot does not start after Composer Upgrade due to Build error

After upgrading to Composer the upgrade Bot does not start.

![image](https://user-images.githubusercontent.com/31212469/117968178-34268180-b326-11eb-9c37-7866b4d7af4d.png)

Below is the error message:

C:\Users\xxx\Documents\Composer\VoiceBot-Demo-Composer\basebot_new\basebot_new\basebot_new.csproj : warning NU1701: Das Paket "Microsoft.Azure.KeyVault.Core 1.0.0" wurde nicht mit dem Projektzielframework ".NETCoreApp,Version=v3.1", sondern mit ".NETFramework,Version=v4.6.1, .NETFramework,Version=v4.6.2, .NETFramework,Version=v4.7, .NETFramework,Version=v4.7.1, .NETFramework,Version=v4.7.2, .NETFramework,Version=v4.8" wiederhergestellt. Dieses Paket ist mit Ihrem Projekt m�glicherweise nicht vollst�ndig kompatibel.
C:\Users\xxx\Documents\Composer\VoiceBot-Demo-Composer\basebot_new\basebot_new\basebot_new.csproj : warning NU1701: Das Paket "Microsoft.Azure.KeyVault.Core 1.0.0" wurde nicht mit dem Projektzielframework ".NETCoreApp,Version=v3.1", sondern mit ".NETFramework,Version=v4.6.1, .NETFramework,Version=v4.6.2, .NETFramework,Version=v4.7, .NETFramework,Version=v4.7.1, .NETFramework,Version=v4.7.2, .NETFramework,Version=v4.8" wiederhergestellt. Dieses Paket ist mit Ihrem Projekt m�glicherweise nicht vollst�ndig kompatibel.
Error occurred building the bot

Unhandled exception. System.ArgumentNullException: Value cannot be null. (Parameter 'containerName') at Microsoft.Bot.Builder.Azure.Blobs.BlobsTranscriptStore..ctor(String dataConnectionString, String containerName, JsonSerializer jsonSerializer) at Microsoft.Bot.Builder.Dialogs.Adaptive.Runtime.Extensions.ServiceCollectionExtensions.AddBotRuntimeTranscriptLogging(IServiceCollection services, IConfiguration configuration) at Microsoft.Bot.Builder.Dialogs.Adaptive.Runtime.Extensions.ServiceCollectionExtensions.AddBotRuntime(IServiceCollection services, IConfiguration configuration) at basebot_new.Startup.ConfigureServices(IServiceCollection services) in C:\Users\xxx\Documents\Composer\VoiceBot-Demo-Composer\basebot_new\basebot_new\Startup.cs:line 24 at System.RuntimeMethodHandle.InvokeMethod(Object target, Object[] arguments, Signature sig, Boolean constructor, Boolean wrapExceptions) at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture) at Microsoft.AspNetCore.Hosting.ConfigureServicesBuilder.InvokeCore(Object instance, IServiceCollection services) at Microsoft.AspNetCore.Hosting.ConfigureServicesBuilder.<>c__DisplayClass9_0.<Invoke>g__Startup|0(IServiceCollection serviceCollection) at Microsoft.AspNetCore.Hosting.ConfigureServicesBuilder.Invoke(Object instance, IServiceCollection services) at Microsoft.AspNetCore.Hosting.ConfigureServicesBuilder.<>c__DisplayClass8_0.<Build>b__0(IServiceCollection services) at Microsoft.AspNetCore.Hosting.GenericWebHostBuilder.UseStartup(Type startupType, HostBuilderContext context, IServiceCollection services) at Microsoft.AspNetCore.Hosting.GenericWebHostBuilder.<>c__DisplayClass12_0.<UseStartup>b__0(HostBuilderContext context, IServiceCollection services) at Microsoft.Extensions.Hosting.HostBuilder.CreateServiceProvider() at Microsoft.Extensions.Hosting.HostBuilder.Build() at basebot_new.Program.Main(String[] args) in C:\Users\xxx\Documents\Composer\VoiceBot-Demo-Composer\basebot_new\basebot_new\Program.cs:line 13

--- a/Composer/packages/server/src/services/project.ts
+++ b/Composer/packages/server/src/services/project.ts
@@ -537,7 +537,10 @@ export class BotProjectService {
                 ? { voiceFontName: 'en-US-AriaNeural', fallbackToTextForSpeechIfEmpty: true }
                 : undefined,
               blobTranscript: originalProject.settings?.blobStorage?.connectionString
-                ? originalProject.settings?.blobStorage
+                ? {
+                    connectionString: originalProject.settings.blobStorage.connectionString,
+                    containerName: originalProject.settings.blobStorage.container,
+                  }
                 : {},
             },
             telemetry: {

